Tasks/PublishTestResults/task.json

```diff
@@ -112,7 +112,6 @@
         }
     },
     "messages": {
-        "mergeFiles": "Number of results files is greater than %d. Therefore merging all files to publish results against a single test run.",
```

Comment:
bump up task version and rebuild the task.

Reply:
Changed it, please check.